iceoryx_dds/include/iceoryx_dds/gateway/iox_to_dds.hpp

```diff
@@ -26,22 +27,33 @@ namespace iox
 {
 namespace dds
 {
-///
 /// @brief DDS Gateway implementation for the iceoryx to DDS direction.
-///
 template <typename channel_t = gw::Channel<popo::UntypedSubscriber, dds::data_writer_t>,
           typename gateway_t = gw::GatewayGeneric<channel_t>>
 class Iceoryx2DDSGateway : public gateway_t
 {
   public:
     Iceoryx2DDSGateway() noexcept;
+
+    /// @brief Load the provided gateway configuration; a channel is setup if required (no error check on channel setup)
```

Comment:
please also move the `(no error ...` to a `@note` tag

Reply:
Move to base class and fixed.